spec/features/case_information_feature_spec.rb

```diff
@@ -1,34 +1,458 @@
+
+        page.execute_script("document.getElementsByName('input-autocomplete')[0].value='NPS - England 3'")
+
+        expect {
+          click_button 'Update'
+        }.to change(enqueued_jobs, :size).by(0)
+
+        expect(page).not_to have_css(".notification")
+        expect(page).not_to have_css(".alert")
+        expectations(probation_service: 'England', tier: 'B', team: 'NPS - England 3', case_allocation: 'NPS')
+      end
+
+      it 'will send an email when probation_service changed to England or Wales from Scotland or Northern Ireland',
+         :raven_intercept_exception,
+         vcr: { cassette_name: :case_information_update_email_sent_when_probation_changed_from_scot_or_ni } do
+        choose_country(country: 'Northern Ireland')
+        click_button 'Continue'
+        visit edit_prison_case_information_path('LEI', nomis_offender_id)
+
+        choose_country(country: 'Wales')
+        second_page_form(case_alloc: 'crc', tier: 'b', team_option: 'NPS - Wales')
+
+        email_expectations(button_name: 'Update',
+                           emails: %w[WalesNPS@example.com spo_user@digital-justice.uk],
+                           spo_notice: 'This is a copy of the email sent to the LDU for your records',
+                           expect_count: :twice)
+
+        within ".notification" do
+          expect(page).to have_content(notification_with_email)
+        end
+
+        expect(page).not_to have_css(".alert")
+        expectations(probation_service: 'Wales', tier: 'B', team: 'NPS - Wales', case_allocation: 'CRC')
+      end
+
+      it 'will send an email when updating team which has a different ldu', :raven_intercept_exception,
+         vcr: { cassette_name: :case_information_update_email_sent_when_updated_team_has_different_ldu } do
+        choose_country(country: 'Wales')
+        click_button 'Continue'
+        second_page_form(case_alloc: 'crc', tier: 'd', team_option: 'NPS - Wales')
+        click_button 'Continue'
+
+        visit edit_prison_case_information_path('LEI', nomis_offender_id)
+
+        choose_country(country: 'England')
+        second_page_form(case_alloc: 'nps', tier: 'a', team_option: 'NPS - England 3')
+
+        email_expectations(button_name: 'Update',
+                           emails: %w[EnglishNPS@example.com spo_user@digital-justice.uk],
+                           spo_notice: 'This is a copy of the email sent to the LDU for your records',
+                           expect_count: :twice)
+
+        within ".notification" do
+          expect(page).to have_content(notification_with_email)
+        end
+
+        expect(page).not_to have_css(".alert")
+        expectations(probation_service: 'England', tier: 'A', team: 'NPS - England 3', case_allocation: 'NPS')
+      end
+    end
   end
 
   it "clicking back link after viewing prisoner's case information, returns back the same paginated page",
+     :raven_intercept_exception,
```

Comment:
not sure about these raven_intercept_exception tags. I think we should only use them very sparsely in the whole codebase if at all.